src/main/feature-extractor.ts

```diff
@@ -6,6 +6,8 @@ import {
   extract as meydaExtract,
   default as Meyda
 } from "meyda";
+import { Observable, from, of } from "rxjs";
+import { take, bufferCount, map, tap } from "rxjs/operators";
```

Comment:
This file should be it's own commit, and subsequent commits should maybe have one implementation that's observable compatible. I think the observable implementation is more concise and clearer regardless.

Reply:
To clarify: It might be good to remove the original implementation, and have the original two feature extractors use the observable based implementation. First would have to make the observable implementation handle hopping windows (negative and positive offsets). Maybe a separate PR, this could go in a GH issue.

Reply:
Then again, it might not, since the inline worker would now have rxjs code in it even if it's not strictly needed, and would increase the bundle size. To do: figure out how much it would increase the bundle size.